DataFormats/Portable/interface/HostProductAlpaka.h

```diff
@@ -0,0 +1,31 @@
+#ifndef DataFormats_Common_interface_HostProduct_H
+#define DataFormats_Common_interface_HostProduct_H
+
+#include <alpaka/alpaka.hpp>
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+// a heterogeneous unique pointer...
+template <typename T>
+class HostProductAlpaka {
```

Comment:
This "heterogeneous unique pointer" should no longer be needed.
Can you simply replace it simply with a `cms::alpakatools::host_buffer<T>` ?